### PR TITLE
DOC: Add note about Inkscape install on Windows

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

It may be easy to not think about, especially when you already have Inkscape installed...

## PR Checklist

**Tests and Styling**
- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [ ] New features are documented, with examples if plot related.
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

File: doc/devel/dependencies.rst [hunk 1]
```diff
@@ -186,6 +186,12 @@ Required:
 - Ghostscript_ (>= 9.0, to render PDF files)
 - Inkscape_ (to render SVG files)
 
+.. note::
```
Comment (timhoffm) [inline doc/devel/dependencies.rst:189]: Indent so that this is within the Inkscape bullet point.

Comment (oscargus) [reply doc/devel/dependencies.rst]: Fixed and seems to work!

**Comment by oscargus** [comment]
There is potentially a similar issue in the Building the Documentation section, but maybe it is enough to have it in one location?

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
I think this is enough.

**Pull request merged**